app/Http/Controllers/UserProjectsController.php

```diff
@@ -17,20 +17,19 @@ public function show(): Response|View
             $projectable->load('projects');
         }
 
-        if ($user->organization) {
-            $projectable = $user->organization;
-            if ($projectable->isConsultant() || $projectable->isConnector()) {
-                $section = 'contracted';
-            } elseif ($projectable->isParticipant()) {
+        if (in_array($user->context, ['individual', 'organization'])) {
```

Comment:
The array values should be sourced from the UserContext enum like:

```suggestion
        if (in_array($user->context, [UserContext::Individual->value, UserContext::Organization->value])) {
```